src/psi/j/executors/flux.py

```diff
@@ -17,6 +18,7 @@
 from psi.j import JobExecutor, ResourceSpecV1
 
 logger = logging.getLogger(__name__)
+logging.basicConfig(level='DEBUG', force=True, filename='flux_connector.log')
```

Comment:
Was adding this intentional?  Looks kinda like a dev-time debug line.

Reply:
Ack, will remove.  @hategan : what would be a proper way to configure logging in `psi-j`?

Reply:
We'll configure it at the library level, so probably there shouldn't be config calls in non top-level files.